src/prepare-app-options.test.ts

```diff
@@ -26,7 +21,7 @@ const defaultPreparedOptions: CaptureScreenShotOptions = {
   debug: false,
   width: 1024,
   height: 1024,
-  inputPath: 'http://localhost:8081/some_model.glb',
```

Comment:
This should have never been a url

Reply:
Correct. I'd actually like to change the return to be `inputUrl` or `modelUrl` or something like that